Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -723,6 +799,38 @@ def get_team_aad_id(team_name: str) -> str:
     raise ValueError('Could not find requested team.')
 
 
+def get_chat_id(chat: str) -> str:
```

Comment:
The name `chat` is a bit ambiguous, even with the explanation.
Perhaps `chat_identifier`?